static/usage.html

```diff
@@ -1021,6 +1022,52 @@ <h2><a href="#app-link-verification">App link verification</a></h2>
                 <a href="https://developer.android.com/training/app-links/verify-site-associations">developer
                 documentation on app link verification</a>.</p>
             </section>
+
+            <section id="carrier-functionality">
+                <h2><a href="#carrier-functionality">Carrier functionality</a></h2>
+
+                <p>GrapheneOS aims to work with all carriers that are officially supported by Google
+                on the stock operating system on Pixel devices. Wi-Fi Calling, VoLTE, Visual Voicemail,
+                MMS, SMS, Calling, 5G (SA and NSA) all are supported however some functionality may 
+                not be usable due to Google not supporting carriers on the stock OS officially or
+                due to GrapheneOS not shipping proprietary apps required in order for this
+                functionality to work on some carriers. GrapheneOS extracts CarrierConfigs, APNs,
+                modem configurations, Visual Voicemail configurations and MMS configurations from the 
+                stock operating system to ensure users get easy carrier support that "just works".</p>
+
+                <p>Please note that in some regions, LTE is referred to as 4G.</p>
+
+                <p>Generally Visual Voicemail, SMS, MMS, Calls and VoLTE will work fine on GrapheneOS 
+                on officially supported carriers by Google. Wi-Fi calling may vary due to a reliance 
+                on proprietary Google apps which GrapheneOS does not ship.</p>
+
+                <p>If you are having problems sending or receiving SMS/MMS messages, we suggest that 
+                you perform the following steps:</p>
+
+                <ul>
+                    <li>Deregister your phone number from <a href="https://selfsolve.apple.com/deregister-imessage">Apple iMessage</a></li>
+                    <li>Deregister your phone number from <a href="https://messages.google.com/disable-chats">Google Chat Features</a></li>
```

Comment:
Trailing s at the end of the link.

Correct link: https://messages.google.com/disable-chat